whois-api/src/main/java/net/ripe/db/whois/api/rest/WhoisServletDeployer.java

```diff
@@ -72,6 +73,7 @@ public WhoisServletDeployer(final WhoisRestService whoisRestService,
                                 final BatchUpdatesService batchUpdatesService,
                                 final HealthCheckService healthCheckService,
                                 final HttpsBasicAuthFiler httpsBasicAuthFiler,
```

Comment:
should we remove this? `HttpsBasicAuthFiler`

Reply:
see above comment